### ISSUE TITLE
chained == comparisons don't work

### ISSUE THREAD

**Issue by tpapp** [jw3126/ArgCheck.jl#6]
```julia
julia> using ArgCheck

julia> x = y = z = 1
1

julia> x == y == z
true

julia> @argcheck x == y == z
ERROR: MethodError: objects of type Int64 are not callable
```

**Comment by jw3126** [comment]
Thanks for reporting! I will fix it at some point. As a workaround, you can of course do
```julia
@argcheck x == y
@argcheck y == z
```

**Comment by tpapp** [comment]
Thanks. Works fine, could you please tag a release with this fix? Eg `v0.2.1`.
